JamJar/EnterConcertInformationController.swift

```diff
@@ -238,7 +270,7 @@ class EnterConcertInformationViewController: BaseViewController, UITextFieldDele
     // Keep track of video URLs being stored
     func callback()
     {
-        self.queue--
+        self.queue -= 1
```

Comment:
Take a look at the **second** answer in this stack overflow jawn - http://stackoverflow.com/questions/11909629/waiting-until-two-async-blocks-are-executed-before-starting-another-block
You could (unlikely, but possible) run into race conditions doing this this way